src/components/learner-credit-management/AssignmentTableRemind.jsx

```diff
@@ -5,17 +5,35 @@ import { Mail } from '@edx/paragon/icons';
 import useRemindContentAssignments from './data/hooks/useRemindContentAssignments';
 import RemindAssignmentModal from './RemindAssignmentModal';
 
-const AssignmentTableRemindAction = ({ selectedFlatRows }) => {
+const calculateTotalToRemind = ({
+  assignmentUuids,
+  isEntireTableSelected,
+  tableItemCount,
+}) => {
+  if (isEntireTableSelected) {
+    return tableItemCount;
```

Comment:
Not all rows/records for the assignment configuration are remindable (only `learner_state="waiting"` is remindable). When the entire table is selected here, we should likely rely on the aggregate counts for each `learner_state` provided by the assignments list API.

```json
{
    "next": "http://enterprise-access.edx.org/api/v1/assignment-configurations/c4315362-0620-4bf9-8ee8-ffcd4e4693c1/admin/assignments/?ordering=-recent_action_time&page=2&page_size=25&state__in=allocated%2Cerrored",
    "previous": null,
    "count": 91,
    "num_pages": 4,
    "current_page": 1,
    "start": 0,
    "results": [...],
    "learner_state_counts": [
        {
            "learner_state": "notifying",
            "count": 51
        },
        {
            "learner_state": "waiting",
            "count": 23
        },
        {
            "learner_state": "failed",
            "count": 17
        }
    ]
}
```

Check the `statusFilterChoices` variable in `BudgetAssignmentsTable` for where these data come into the component.

Reply:
Ah, right, thanks for the reminder about that!